tracker/event_tracker.go

```diff
@@ -44,19 +46,20 @@ func (e *EventTracker) Start(ctx context.Context) error {
 		return err
 	}
 
-	store, err := boltdbStore.New(e.dbPath)
+	notifierCh := make(chan []*ethgo.Log)
 
+	store, err := NewEventTrackerStore(e.dbPath, e.finalityDepth, notifierCh, e.logger)
 	if err != nil {
 		return err
 	}
 
-	e.logger.Info("Start tracking events", "contract address", e.contractAddr)
+	e.logger.Info("Start tracking events", "block finality depth", e.finalityDepth, "contract address", e.contractAddr)
 
 	tt, err := tracker.NewTracker(provider.Eth(),
 		tracker.WithBatchSize(10),
 		tracker.WithStore(store),
 		tracker.WithFilter(&tracker.FilterConfig{
-			Async: false,
+			Async: true,
```

Comment:
Heads up:
this was changed a couple of months ago because it was noticed that putting the tracker in async mode, resulted in gaps in captured logs (bridge e2e tests were failing occasionally).
If you were able to overcome such a problem with custom-made storage, then we are ok.

Reply:
Old channel from tracker is not used anymore. By setting this configuration to true, we are preventing blocking of tracker. Otherwise we need to (dummy)read that old channel. It is easier this way.